examples/tree.rs

```diff
@@ -107,8 +107,12 @@ impl<ID: TreeId, TM: TreeMeta, A: Actor + std::fmt::Debug> Replica<ID, TM, A> {
         // is the causally stable threshold.
 
         let mut v: Vec<&Clock<A>> = self.latest_time_by_replica.values().collect();
-        v.sort_unstable_by(|a, b| a.cmp(b));
-        v.pop() 
+        v.sort();
```

Comment:
What do you think of 
`vec.sort();
vec.reverse():
vec.pop
`?

Reply:
should work, but depending on Vec's implementation it could be hugely more expensive on a large list. So I'd prefer not to unless its shown/known that Vec's reverse() is fast.  I don't see any mention in the reverse() docs.

a couple notes:

1. VecDeque may be better suited for this, as it offers push_back and pop_front to use the start of list.
2. Unless I'm missing something the LogOpMove Vec could actually store at end instead of beginning.  I only did it this way because that's what they specify in the paper and I tried to follow their algo as exactly as I could.

I was actually rather surprised Vec doesn't have shift/unshift or equivalent for working with front of array.  Other langs/libs generally support this.

Reply:
So I think this is faster on large lists:

```
        v.sort_by(|a, b| 
            match a.cmp(b) {
                Ordering::Greater => Ordering::Less,
                Ordering::Equal => Ordering::Equal,
                Ordering::Less => Ordering::Greater,
            }
        );
```

but it is also much uglier, and getting carried away.  :)

Really this is just sorting replicas, and it is just an example, so upon reflection I'm good with 'vec.sort(); vec.reverse(): vec.pop' for readability.

Reply:
pushed.